javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue128.java

```diff
@@ -30,7 +31,7 @@
     @Before
     public void setup() throws IOException {
         Path srcDir = adaptPath("src/test/resources/issue128");
-        typeSolver = new CombinedTypeSolver(new ReflectionTypeSolver(), new JavaParserTypeSolver(srcDir));
+        typeSolver = new CombinedTypeSolver(new ReflectionTypeSolver(), new JavaParserTypeSolver(srcDir, new LeanParserConfiguration()));
```

Comment:
Should we use singleton for `LeanParserConfiguration`? Or for that we would need to make it immutable?

Reply:
That would be extra code, and now we're wasting maybe a few bytes in a testrun that takes gigabytes :-)